portafly/src/components/pages/accounts/utils/accountsTableDataFactory.tsx

```diff
@@ -1,18 +1,21 @@
 import React from 'react'
 import { sortable } from '@patternfly/react-table'
-import { IDeveloperAccount } from 'types'
-import { TFunction } from 'i18next'
-import { ActionButtonImpersonate } from 'components/pages/accounts'
+import { IDeveloperAccount, DataListRowGenerator, DataListColumnGenerator } from 'types'
+import { ActionButtonImpersonate, AccountOverviewLink } from 'components/pages/accounts'
 
-const generateRows = (accounts: IDeveloperAccount[]) => {
+const generateRows: DataListRowGenerator = (accounts: IDeveloperAccount[]) => {
   const isMultitenant = process.env.REACT_APP_MULTITENANT
   // Rows and Columns must have the same order
   const mapAccountToRowCell = (account: IDeveloperAccount) => [
-    account.org_name,
+    {
+      filterableString: account.org_name,
```

Comment:
This is up to interpretation, but what I get from `filterableString` (the construction of the adj + string) is that one can (is able to) filter that particular string, that string is filterable. While something like `filterString` (noun + string) is that string _is_ the filter string, it's used for filtering purposes. 
Again, it's interpretation... Maybe someone else can give us some feedback as well.

Reply:
I agree with you, I don't like the name and have been looking for a better one. I thought about implementing `toString` instead:
```suggestion
      toString: () => account.org_name,
```
Other options `computedString`, `stringValue`, `value`....